supplementary_style_guide/style_guidelines/graphical-interfaces.adoc

```diff
@@ -29,7 +29,9 @@ In the *Name* field, enter `test-postgresql`.
 [[user-interface-elements]]
 == User interface elements
 
-Bold all graphical user interface elements, including, but not limited to, menu names, menu items, button names, dialog box names, and window names. Bold the item, even if it is not clickable.
+Use bold text for all graphical user interface (GUI) element names, including, but not limited to, menu, menu item, button, dialog box, and window names. Use bold text for the element name if the name appears in the GUI, even if the element is not clickable.
+
+If an element is not labeled in the GUI, refer to the element by a generic description and do not use bold text. For example, if a search field is not labeled in the GUI, write it as "the search field", not "the *Search* field".
 
 .Example AsciiDoc
```

Comment:
You might want to move the Example so that it appears after the first paragraph (Use bold text...)

If you want to add an example to the new paragraph, you could use "Click the edit icon." or "Enter the term in the search field."